src/Components/PropertyInspector/PropertyInspectoryModel.ts

```diff
@@ -646,53 +650,61 @@ abstract class PropertyInspectorModel {
         }
     };
 
-    static conformDtdlInterface = (model: DtdlInterface) => {
-        if (!model) return null;
-        const conformedModel = JSON.parse(JSON.stringify(model));
+    static replaceKeyInObj = (obj, oldKey, newKey) => {
+        delete Object.assign(obj, { [newKey]: obj[oldKey] })[oldKey];
+    };
 
-        const replaceKeyInObj = (obj, oldKey, newKey) => {
-            delete Object.assign(obj, { [newKey]: obj[oldKey] })[oldKey];
-        };
+    static conformSyntax = (inputEl) => {
+        // Update syntax of all object keys
+        if (typeof inputEl === 'object') {
```

Comment:
inputEl might be `null` which is also an object type. not sure if that would be a case but maybe handling null might be good before passing it to Object.keys.

Reply:
done!